_includes/configuration/thread_pools.html

```diff
@@ -7,10 +7,6 @@
 Configuring Collins akka pools is relatively straightforward. See the akka documentation for details. Collins makes use of a single akka dispatcher which can be configured in <code>akka.actor.default-dispatcher</code>. The dispatcher is shared for background processes and solr updates
 </p>
 
-<p>
-<span class="label label-note">Note</span> Currently only on the development branch, scheduled for future release.
```

Comment:
Might still be good to leave this in, as the feature is still not in the stable release.  We'll just have to remember to circle back when we do cut the release and remove it then. That said, our docs are already out of sync with the releases, so I'm happy either way.